x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/util/UnsignedLongUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.ql.util;
+
+import java.math.BigInteger;
+
+public abstract class UnsignedLongUtils {
```

Comment:
Instead of using a dedicated utils just for unsigned longs, it would make sense to create a util class for number/numerics. So methods spread around such as  DataTypeConver#safeTo could be moved here
This can be a follow-up PR however for now I would rename this to NumericUtils.

Reply:
Makes sense, changed.